Set cache mode for target installation disk to unsafe for better
performance.
Index: virt-manager-5.1.0/virtinst/install/installer.py
===================================================================
--- virt-manager-5.1.0.orig/virtinst/install/installer.py
+++ virt-manager-5.1.0/virtinst/install/installer.py
@@ -572,6 +572,15 @@ class Installer:
 
     def _build_postboot_xml(self, guest_ro, final_xml, meter):
         initial_guest = Guest(self.conn, parsexml=final_xml)
+        # At install time set the target disk to 'unsafe' for
+        # better performance if the target is not a block device
+        saved_cache = "None"
+        if initial_guest.devices.disk:
+            target_disk = initial_guest.devices.disk[0]
+            saved_cache = target_disk.driver_cache
+            if target_disk.type != DeviceDisk.TYPE_BLOCK and target_disk.driver_io != "native":
+                target_disk.driver_cache = DeviceDisk.CACHE_MODE_UNSAFE
+
         self._alter_bootconfig(initial_guest)
         self._alter_install_resources(initial_guest, meter)
         if self.has_cloudinit():
@@ -601,11 +610,15 @@ class Installer:
                 )
                 initial_guest.remove_device(initial_guest.devices.tpm[0])
 
+        install_xml = initial_guest.get_xml()
+        if saved_cache != "None":
+            target_disk.driver_cache = saved_cache
+
         final_guest = Guest(self.conn, parsexml=final_xml)
         self._remove_install_cdrom_media(final_guest)
         self._remove_unattended_install_cdrom_device(final_guest)
 
-        return initial_guest.get_xml(), final_guest.get_xml()
+        return install_xml, final_guest.get_xml()
 
     def _build_xml(self, guest, meter):
         initial_xml = None
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-url.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cdrom-url.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-url.xml
@@ -24,6 +24,7 @@
       <target dev="hda" bus="ide"/>
       <readonly/>
       <boot order="1"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-memory-hotplug.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-memory-hotplug.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-memory-hotplug.xml
@@ -33,7 +33,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2" discard="unmap"/>
+      <driver name="qemu" type="qcow2" discard="unmap" cache="unsafe"/>
       <source file="/var/lib/libvirt/images/fedora.qcow2"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-double.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cdrom-double.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-double.xml
@@ -21,6 +21,7 @@
       <source file="/var/lib/libvirt/images/vm1.qcow2"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
+      <driver cache="unsafe"/>
     </disk>
     <disk type="file" device="cdrom">
       <source file="/pool-dir/testvol1.img"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-default.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-default.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-default.xml
@@ -33,6 +33,7 @@
       <driver name="qemu" type="qcow2"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="qemu-xhci" ports="15"/>
     <controller type="pci" model="pcie-root"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options1.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-options1.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options1.xml
@@ -44,6 +44,7 @@ chpasswd:
       <driver name="qemu" type="qcow2"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="qemu-xhci" ports="15"/>
     <controller type="pci" model="pcie-root"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options2.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-options2.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options2.xml
@@ -39,6 +39,7 @@ ssh_authorized_keys:
     <disk type="file" device="disk">
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options3.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-options3.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options3.xml
@@ -33,6 +33,7 @@ users:
     <disk type="file" device="disk">
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options4.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-options4.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options4.xml
@@ -26,6 +26,7 @@
     <disk type="file" device="disk">
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options5.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cloud-init-options5.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cloud-init-options5.xml
@@ -26,6 +26,7 @@
     <disk type="file" device="disk">
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-multiple-short-id.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-osinfo-multiple-short-id.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-multiple-short-id.xml
@@ -29,7 +29,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
@@ -32,7 +32,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
     </disk>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-win7-unattended.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-osinfo-win7-unattended.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-win7-unattended.xml
@@ -44,7 +44,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="cdrom">
-      <driver name="qemu"/>
+      <driver name="qemu" cache="unsafe"/>
       <source file="TESTSUITE_SCRUBBED/tests/data/fakemedia/fake-win7.iso"/>
       <target dev="sda" bus="sata"/>
       <readonly/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-osvariant-defaults-pxe.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-osvariant-defaults-pxe.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-osvariant-defaults-pxe.xml
@@ -27,6 +27,7 @@
       <source file="/var/lib/libvirt/images/fedora26.qcow2"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-reinstall-location.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-reinstall-location.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-reinstall-location.xml
@@ -22,7 +22,7 @@
   <devices>
     <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
     <disk type="file" device="disk">
-      <driver type="qcow2"/>
+      <driver type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/test-clone-simple.img"/>
       <target dev="hda" bus="ide"/>
       <address type="drive" controller="0" bus="0" target="0" unit="0"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-reinstall-pxe.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-reinstall-pxe.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-reinstall-pxe.xml
@@ -19,7 +19,7 @@
   <devices>
     <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
     <disk type="file" device="disk">
-      <driver type="qcow2"/>
+      <driver type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/test-clone-simple.img"/>
       <target dev="hda" bus="ide"/>
       <address type="drive" controller="0" bus="0" target="0" unit="0"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-w2k3-cdrom.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-w2k3-cdrom.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-w2k3-cdrom.xml
@@ -29,6 +29,7 @@
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
+      <driver cache="unsafe"/>
     </disk>
     <disk type="file" device="cdrom">
       <source file="/pool-dir/testvol2.img"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-aarch64-cdrom.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-aarch64-cdrom.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-aarch64-cdrom.xml
@@ -24,7 +24,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-aarch64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-centos-label.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-cdrom-centos-label.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-cdrom-centos-label.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-centos7.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-centos7.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-centos7.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-cpu-default-fallback.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-cpu-default-fallback.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-cpu-default-fallback.xml
@@ -32,7 +32,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-cpu-hostmodel-fallback.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-cpu-hostmodel-fallback.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-cpu-hostmodel-fallback.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-fedoralatest-url.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-fedoralatest-url.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-fedoralatest-url.xml
@@ -33,7 +33,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
     </disk>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel5.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-rhel5.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel5.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel6.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-rhel6.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel6.xml
@@ -33,7 +33,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
     </disk>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel7.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-rhel7.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-rhel7.xml
@@ -33,7 +33,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
     </disk>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-session-defaults.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-session-defaults.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-session-defaults.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2" discard="unmap"/>
+      <driver name="qemu" type="qcow2" discard="unmap" cache="unsafe"/>
       <source file="/tmp/.local/share/libvirt/images/fedora21.qcow2"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-win10.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-win10.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-win10.xml
@@ -45,7 +45,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="sda" bus="sata"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-win2k3-cdrom.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-kvm-win2k3-cdrom.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-kvm-win2k3-cdrom.xml
@@ -45,7 +45,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-linux2020.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-linux2020.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-linux2020.xml
@@ -30,7 +30,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2" discard="unmap"/>
+      <driver name="qemu" type="qcow2" discard="unmap" cache="unsafe"/>
       <source file="/var/lib/libvirt/images/linux2020.qcow2"/>
       <target dev="vda" bus="virtio"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-location-iso.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-location-iso.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-location-iso.xml
@@ -32,7 +32,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="vda" bus="virtio"/>
     </disk>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-location-manual-kernel.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-location-manual-kernel.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-location-manual-kernel.xml
@@ -27,7 +27,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="cdrom">
-      <driver name="qemu"/>
+      <driver name="qemu" cache="unsafe"/>
       <source file="TESTSUITE_SCRUBBED/tests/data/fakemedia/fake-no-osinfo.iso"/>
       <target dev="hda" bus="ide"/>
       <readonly/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-netinst-unattended.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-osinfo-netinst-unattended.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-osinfo-netinst-unattended.xml
@@ -32,7 +32,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="cdrom">
-      <driver name="qemu"/>
+      <driver name="qemu" cache="unsafe"/>
       <source file="TESTSUITE_SCRUBBED/tests/data/fakemedia/fake-f26-netinst.iso"/>
       <target dev="sda" bus="sata"/>
       <readonly/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-q35-defaults.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-q35-defaults.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-q35-defaults.xml
@@ -25,7 +25,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="sda" bus="sata"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-remote-storage.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-remote-storage.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-remote-storage.xml
@@ -21,6 +21,7 @@
       <source file="/foo/bar/baz"/>
       <target dev="hda" bus="ide"/>
       <boot order="2"/>
+      <driver cache="unsafe"/>
     </disk>
     <disk type="block" device="disk">
       <source dev="/dev/zde"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-unattended-remote-cdrom.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-unattended-remote-cdrom.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-unattended-remote-cdrom.xml
@@ -27,6 +27,7 @@
       <target dev="hda" bus="ide"/>
       <readonly/>
       <boot order="1"/>
+      <driver cache="unsafe"/>
     </disk>
     <controller type="usb" model="ich9-ehci1"/>
     <controller type="usb" model="ich9-uhci1">
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-win7-uefi.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-win7-uefi.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-win7-uefi.xml
@@ -47,7 +47,7 @@
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="sda" bus="sata"/>
       <boot order="2"/>
Index: virt-manager-5.1.0/tests/data/cli/compare/virt-install-xen-pv.xml
===================================================================
--- virt-manager-5.1.0.orig/tests/data/cli/compare/virt-install-xen-pv.xml
+++ virt-manager-5.1.0/tests/data/cli/compare/virt-install-xen-pv.xml
@@ -16,7 +16,7 @@
   </os>
   <devices>
     <disk type="file" device="disk">
-      <driver name="qemu" type="qcow2"/>
+      <driver name="qemu" type="qcow2" cache="unsafe"/>
       <source file="/pool-dir/testvol1.img"/>
       <target dev="xvda" bus="xen"/>
     </disk>
